src/UI-Constants/Color.yaml

```diff
@@ -0,0 +1,42 @@
+COLOR_BACKGROUND: '#ecf0f1'
```

Comment:
Do we really need so many colors? I think we should keep only those which we are going to use as per the UI designs.

Reply:
Actually I don't have complete UI design of the app, and is currently being designing. Keeping all this in mind added all this colors that can be used as per need later.

Reply:
Ok anyways, we can update it on the go.